src/content/language/operators.md

```diff
@@ -14,29 +14,29 @@ nextpage:
-| logical OR                              | `|               |`                                                                                                                                                                                              | Left          |
-| if null                                 | `??`                                                                                                                                                                                              | Left          |
-| conditional                             | <code><em>expr1</em> ? <em>expr2</em> : <em>expr3</em></code>                                                                                                                                     | Right         |
-| cascade                                 | `..` &nbsp;&nbsp; `?..`                                                                                                                                                                           | Left          |
-| assignment                              | `=`    `*=`    `/=`    `+=`    `-=`    `&=`    `^=`    <em>etc.</em>                                                                                                                              | Right         |
+| multiplicative                          | `*`    `/`    `%`  `~/`                                                                                                                                                                            | Left          |
+| additive                                | `+`    `-`                                                                                                                                                                                         | Left          |
+| shift                                   | `<<`    `>>`    `>>>`                                                                                                                                                                              | Left          |
+| bitwise AND                             | `&`                                                                                                                                                                                                | Left          |
+| bitwise XOR                             | `^`                                                                                                                                                                                                | Left          |
+| bitwise OR                              | `                                                                                                                                                                                                  | `             | Left |
```

Comment:
It looks like this entry will need to be fixed as well to: `<code>&#124;</code>

Reply:
Had some issues with previous branch. I have opened a new pull request. Can you review that?